internal/guest/runtime/runc/runc.go

```diff
@@ -139,7 +144,7 @@ func (c *container) Start() error {
 	if err != nil {
 		runcErr := getRuncLogError(logPath)
 		c.r.cleanupContainer(c.id)
-		return errors.Wrapf(err, "runc start failed with %v: %s", runcErr, string(out))
+		return errors.Wrapf(runcErr, "runc start failed with %v: %s", err, string(out))
```

Comment:
Wait I think it was right the original way, as the format string is stating that runc failed with %v and is supposed to get replaced with whatever was found in the Runc log file which runcErr has the contents of

Reply:
Although maybe I'm missing something

Reply:
because of `Wrapf`, both will get formatted into the string when printed, but `err` is usually a generic "runc command failed". Wrapping the runc log file error allows the use of `errors.Is(`, vs having to search through the error string for whatever the log file message was.